conmon-rs/common/proto/conmon.capnp

```diff
@@ -45,4 +45,16 @@ interface Conmon {
     }
 
     execSyncContainer @2 (request: ExecSyncContainerRequest) -> (response: ExecSyncContainerResponse);
+
+    ###############################################
+    # Attach
+    struct AttachContainerRequest {
```

Comment:
We could either add an optional ExecSession parameter here, or add a different endpoint. I am open to either. If we do the former, we may just want to call this "AttachRequest" as it'll be for container or exec

Reply:
I added a new `execSessionID` parameter to the request and renamed it to `AttachRequest/Response`